src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        int randomIndex = new Random().nextInt(colors.length);
+        return colors[randomIndex].toString();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/6f0ba76d-0fdf-469e-a973-62c7441decd3)

Reply:
not fixed